docs/swagger.yaml

```diff
@@ -362,13 +373,27 @@ components:
           maxLength: 255
           deprecated: true
           example: posstandard
-          description: (deprecated) Optional. One of the price packages that are registered for the partner in the Vipps system. Please note, it is not the PriceId (3 digit), but the name, such as "standard", "price1", etc, that is applicable to the partner. If the field is not provided, the merchant must select it manually from the product order section of the merchant portal before submitting. 
+          description: >-
+            (deprecated) Optional.
+            One of the price packages that are registered for the partner in the
```

Comment:
```suggestion
            One of the price packages that is registered for the partner in the
```

Reply:
That one is a bit tricky. https://www.aje.com/arc/editing-tip-subject-verb-agreement-across-prepositional-phrases/ I felt by instinct that is was "is", but the prepositional phrase complicates figuring out which is the actual subject.